pkg/controllers/management/kontainerdrivermetadata/data.go

```diff
@@ -328,6 +360,85 @@ func (md *MetadataController) createOrUpdateAddonCRD(addonName, template string,
 	return nil
 }
 
+func (md *MetadataController) createOrUpdateCisConfigCRD(
+	k8sVersion string,
+	cisConfigParams v3.CisConfigParams,
+) error {
+	cisConfig, err := md.CisConfigLister.Get(namespace.GlobalNamespace, k8sVersion)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			return fmt.Errorf("error finding cisConfig for k8sVersion: %v", k8sVersion)
+		}
+		cisConfig = &v3.CisConfig{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       v3.CisConfigGroupVersionKind.Kind,
+				APIVersion: v3.SchemeGroupVersion.String(),
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      k8sVersion,
+				Namespace: namespace.GlobalNamespace,
+			},
+			Params: cisConfigParams,
+		}
+		logrus.Debugf("driverMetadata: creating cisConfig CRD for k8sVersion: %v",
+			k8sVersion)
+		if _, err := md.CisConfig.Create(cisConfig); err != nil && !errors.IsAlreadyExists(err) {
+			return err
+		}
+		return nil
+	}
+	if reflect.DeepEqual(cisConfig.Params, cisConfigParams) {
+		return nil
+	}
+	logrus.Debugf("driverMetadata: cisConfigParams changed for k8sVersion: %v old: %v new: %v",
+		k8sVersion, cisConfig.Params, cisConfigParams)
+	cisConfigCopy := cisConfig.DeepCopy()
+	cisConfigCopy.Params = cisConfigParams
+	if _, err := md.CisConfig.Update(cisConfigCopy); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (md *MetadataController) createOrUpdateCisBenchmarkVersionCRD(
```

Comment:
nit: CRD should not be in the name of the func

Reply:
I think it's easier to read and understand the code with CRD in place, let's leave it this way :)